src/types/libdef/npm/jest_v22.x.x.js

```diff
@@ -1,5 +1,11 @@
-// flow-typed signature: d5b32a3045854623325a334220fbc654
-// flow-typed version: c7c67b81c1/jest_v20.x.x/flow_>=v0.39.x
+// flow-typed signature: 18018da6c1a1d95b4ab1c64bb5fe86ca
+// flow-typed version: c1ad61e7d4/jest_v22.x.x/flow_>=v0.39.x
```

Comment:
Too much changes in this file because of prettier, I'll revert most of the changes before landing...